data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -0,0 +1,54 @@
+package co.nimblehq.compose.crypto.data.model.response
+
+import co.nimblehq.compose.crypto.domain.model.CoinItem
+import com.squareup.moshi.Json
+
+data class CoinItemResponse(
+    @Json(name = "id")
+    val id: String?,
+    @Json(name = "symbol")
+    val symbol: String?,
+    @Json(name = "name")
+    val coinName: String?,
+    @Json(name = "current_price")
+    val currentPrice: Int?,
+    @Json(name = "market_cap")
+    val marketCap: Long?,
+    @Json(name = "market_cap_rank")
+    val marketCapRank: Int?,
+    @Json(name = "total_volume")
+    val totalVolume: Long?,
+    @Json(name = "high_24h")
+    val high24h: Long?,
+    @Json(name = "low_24h")
+    val low24h: Long?,
+    @Json(name = "price_change_24h")
+    val priceChange24h: Long?,
+    @Json(name = "price_change_percentage_24h")
+    val priceChangePercentage24h: Int?,
+    @Json(name = "market_cap_change_24h")
+    val marketCapChange24h: Long?,
+    @Json(name = "market_cap_change_percentage_24h")
+    val marketCapChangePercentage24h: Int?,
+    @Json(name = "price_change_percentage_24h_in_currency")
+    val priceChangePercentage24hInCurrency: Long?
```

Comment:
@hoangnguyen92dn I didn't see this field for this endpoint `coins/markets` in `https://www.coingecko.com/en/api/documentation` Also, for the `OkHttp` log, Can we remove it 🤔 
```suggestion
    @Json(name = "price_change_percentage_24h_in_currency")
    val priceChangePercentage24hInCurrency: Long?
```

Reply:
@Wadeewee This field response is based on this param on request `price_change_percentage=24h`. You can find `price_change_percentage_24h_in_currency ` on the response log on PR's description. This field will be used to display the price changed on integration task

Reply:
Oh, I see @hoangnguyen92dn . I didn't get it because I requested `price_change_percentage=1h` but after I changed it to `24h` and then I got this field `price_change_percentage_24h_in_currency`.

Reply:
@Wadeewee Hmm. So we should notice about this, the response field could reflect our request.